spec/models/manageiq/providers/cisco_intersight/physical_infra_manager/refresher_spec.rb

```diff
@@ -83,7 +83,7 @@ def assert_specific_physical_server
                         :raw_power_state        => "",
```

Comment:
I would have expected raw_power_state to also change.

Reply:
raw_power_state hasn't changed at all (with this commit: https://github.com/ManageIQ/manageiq-providers-cisco_intersight/pull/41/commits/b14786b0bd1b799230cfeb2025093b129a054ea0), that why it's left the same as it was.

Running the tests (using command `bundle exec rake app:spec:models`) inside our provider also doesn't raise any exception: 

<img width="998" alt="image" src="https://user-images.githubusercontent.com/92620429/165042023-9f9f53ae-b025-4b01-80a7-67036b288714.png">

Reply:
`raw_power_state` is intended to have the "native" value for the power state and `power_state` should have a normalized power state which is consistent across providers.  Lenovo has "On", "Off", "Standby" for example.  If there are more power states that you'd need to represent we can include them in the "generic" set.

> AdminPowerState comes into play when we want to change the state using post/patch methods. Also, changing AdminPowerState automatically changes OperPowerState. In ManageIQ, AdminPowerState is needed to do post/patch operations on.

Typically rather than `PATCH {"raw_power_state": "on"}` to start a server we would `POST {"action": "start"}` so I'm not sure we need both the current and desired power state in our database.

Reply:
Understand, thanks @agrare. Other provider developers from XLAB also agreed on this (@alescernivec). The commit below changes this.